### ISSUE TITLE
Add contact us page.

### ISSUE THREAD

**Issue by rashen33** [rashen33/movie-app-react#5]

**Comment by HarryHeres** [comment]
Hi, I’m new to FE development but got lots of experience as a backend dev, so I would like to enhance my skills in front end. I find this a great opportunity to try out react. Can I work on this?

**Comment by rashen33** [comment]
Hi @HarryHeres . Yes sure you can work on this. I will assign you.

**Comment by Vivek7038** [comment]
can i work on this , i have experience doing this

**Comment by HarryHeres** [comment]
> can i work on this , i have experience doing this

If its too much, I will hand it off to you. Sounds OK?

**Comment by HarryHeres** [comment]
Also @rashen33 seems like the Preview page is not available?

**Comment by rashen33** [comment]
Yeah @HarryHeres  there is a issue with the deployment. I will fix it soon.

**Comment by HarryHeres** [comment]
Perfect, thanks

**Comment by HarryHeres** [comment]
@rashen33 What type of contact info would you like to be included?
